src/utils/code.ts

```diff
@@ -365,6 +365,22 @@ const getIconsImports = (components: IComponents) => {
   })
 }
 
+export const generateMainTsx = (params: any, fileName: string) => {
+  let refsCode = ``
+  let appCode = `return <${fileName}App \n`
+  params.map((param: any) => {
+    appCode += `${param.name}={${param.name}}\n`
+    if (
+      param.name.length >= 4 &&
+      param.name.slice(param.name.length - 4) === '_ref'
```

Comment:
add field in param to check if ref

Reply:
We wont need it if can have a dictionary of all ref types. We can just check for that rather than having an extra isRef property.

Reply:
Would be faster to access and clear with isRef

Reply:
Done in the latest commit